pyanaconda/modules/timezone/timezone.py

```diff
@@ -167,6 +171,20 @@ def set_timezone_with_priority(self, timezone, priority):
         self.timezone_changed.emit()
         log.debug("Timezone is set to %s.", timezone)
 
+    def get_timezones(self):
+        """Get all valid timezones.
```

Comment:
`get_all_valid_timezones` would be better.

Reply:
Indeed, changing the method name. :)

Reply:
Hmm, what about the interface method name ? That would be: `GetAllValidTimezones()` - looks a bit cumbersome. :P What do you think ?

Reply:
Ah, yes, please rename it to match as well. (Mentioned also below in the second round.)